### PR TITLE
P#60739 Remove unsupported data types - Data types still visible in some lists 

### PR DESCRIPTION
**Pull request by vu-vuong-esg**
P#60739 Remove unsupported data types - Data types still visible in some lists 
Fix https://github.com/onOfficeGmbH/oo-wp-plugin/issues/151

### CODE REVIEW

**Comment by hungnc89** [comment]
@jayay could you please review this PR?

**Comment by tang-hien-egs** [comment]
@jayay could you please review this PR?

File: plugin/Field/Collection/FieldLoaderGeneric.php [hunk 1]
```diff
@@ -80,7 +80,7 @@ public function load(): Generator
 			if (isset($fieldArray['label'])) {
 				unset($fieldArray['label']);
 			}
-			$listTypeUnSupported = ['user', 'datei', 'redhint', 'blackhint', 'dividingline'];
+			$listTypeUnSupported = ['user', 'date', 'redhint', 'blackhint', 'dividingline'];
```
Comment (jayay) [inline plugin/Field/Collection/FieldLoaderGeneric.php:83]: Did you check which one is correct? _datei_ is German for _file_, which is what is supposed to be removed in this project.

Comment (jayay) [reply plugin/Field/Collection/FieldLoaderGeneric.php:83]: This also applies to the other changes of the commit

**Review by jayay — APPROVED** [review]

**Review by jayay — DISMISSED** [review]

**Comment by tang-hien-egs** [comment]
@jmaas-onoffice , Jakob approved , could you check PR help me ?

**Comment by jmaas-onoffice** [comment]
This is still in the "old" workflow (marked with the `feedback-jakob` label), so the testers will have a look.

I'm currently busy with the other tickets.

**Comment by tang-hien-egs** [comment]
> 
sorry, i will ask Michael check it help me !

**Comment by tang-hien-egs** [comment]
@michaelarnold-dev , could you check PR help me ?

**Comment by tang-hien-egs** [comment]
@michaelarnold-dev , please check PR help me ?

**Comment by michaelarnold-dev** [comment]
@tang-hien-egs according to Robert Igelmund the code still doesn't work as expacted. Here you can find a screenshot
data_types_still_visible

**Comment by tang-hien-egs** [comment]
> @tang-hien-egs according to Robert Igelmund the code still doesn't work as expacted. Here you can find a screenshot data_types_still_visible

@michaelarnold-dev , could you give me information about serve testing, because i test in local is Ok 
img_test_60739
?

**Review by jmaas-onoffice — APPROVED** [review]

**Review by jmaas-onoffice — CHANGES_REQUESTED** [review]
Could you fix the failing unit test? I merged `master` into here and now some of the tests are failing.

**Comment by vu-vuong-esg** [comment]
> Could you fix the failing unit test? I merged `master` into here and now some of the tests are failing.

i fixed errors, please check PR help me !

**Review by jmaas-onoffice — APPROVED** [review]

**Review by jmaas-onoffice — CHANGES_REQUESTED** [review]
When I add some of those unsupported fields to a list in version 3.1 and then update, the fields stay in the estate list. They should be removed.

Note that they are removed from the fields that you can select on the left, but they remain in the list on the right.

**Comment by tang-hien-egs** [comment]
@jmaas-onoffice : is this your mean like image ?
JOHANNESS_1


JOHANNESS_222

**Comment by jmaas-onoffice** [comment]
That's the correct part of the interface, but I don't know if it is clear what should happen. So here's an example.

Before the update, on version 3.1, I can add the following fields. (The first two are normal fields, the others are of types that this PR should remove.)

grafik

Now I update to [3.1-21-g03bd2867](https://github.com/onOfficeGmbH/oo-wp-plugin/pull/168#issuecomment-1124581277). As you can see in the following image, the fields are gone from the left, but they are still on the right.

grafik

When I update, the fields on the right should be removed so that it looks like this:

grafik

**Comment by tang-hien-egs** [comment]
> That's the correct part of the interface, but I don't know if it is clear what should happen. So here's an example.
> 
> Before the update, on version 3.1, I can add the following fields. (The first two are normal fields, the others are of types that this PR should remove.)
> 
> grafik
> 
> Now I update to [3.1-21-g03bd2867](https://github.com/onOfficeGmbH/oo-wp-plugin/pull/168#issuecomment-1124581277). As you can see in the following image, the fields are gone from the left, but they are still on the right.
> 
> grafik
> 
> When I update, the fields on the right should be removed so that it looks like this:
> 
> grafik

yes, i clean requiremet , i will update it !

**Comment by yeneastgate** [comment]
Convert code to this pull request #258

**Pull request closed without merging**